ipatests/test_integration/test_sudo.py

```diff
@@ -680,3 +693,45 @@ def test_category_all_validation_runasgroup(self):
                                           '--groups', 'testgroup2'],
                                           raiseonerr=False)
         assert result.returncode != 0
+
+
+class TestSudoAdvise(TestSudoBase):
```

Comment:
A new test class means that the test runner also has to install a new server just for these tests. This will prolongate testing by more than 10 minutes.

Reply:
I explained why I did it  in  https://github.com/freeipa/freeipa/pull/2494#issuecomment-433840963
I have to deal with the fact, that existing tests do not make cleanup. So to avoid creating new class I will have to put tests at the top of testclas  and to ask in comments not to add anything before them, which looks a little bit ugly to me.

Reply:
Technically speaking, existing tests perform a cleanup. They use a tactical nuke to tear down the whole cluster with ``ipa-server-install --uninstall`` instead of a surgical scalpel to remove some data. :rofl: